app/views/layouts/_user_menu.html.haml

```diff
@@ -44,3 +44,7 @@
     = link_to(admin_users_path) do
       %span.fa.fa-user
       Users
+%li
+  = link_to(admin_revision_history_path) do
+    %span.fa.fa-history
+    Revision History
```

Comment:
You should only show this if a user has the appropriate access to see the page. Otherwise there will be a link which won't be accessible by the user.